### PR TITLE
fix(ui): Overview page pagination fixes.

### PR DESCRIPTION
**Pull request by shazlehu**
### Proposed Change
Limit configuration and destination tables to 25 per page, make page selection visible on overview page tables. Resolves observIQ/iris#1379

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

**Review by Mrod1598 — APPROVED** [review]

File: ui/src/pages/overview/OverviewPage.tsx [hunk 1]
```diff
@@ -125,17 +125,28 @@ const OverviewPageContent: React.FC = () => {
...
-          parent.innerHTML = "";
-        }
+      for (let i = 0; i < paginationSelectLabels.length; i++) {
+        paginationSelectLabels[i].innerHTML = "";
+      }
+      for (let i = 0; i < paginationSelectLabels.length; i++) {
+        paginationSelects[i].innerHTML = "";
+      }
+      for (let i = 0; i < paginationSelectLabels.length; i++) {
+        rowsSelected[i].innerHTML = "";
```
Comment (armstrmi) [inline ui/src/pages/overview/OverviewPage.tsx:149]: for loop iterator is set to be less than `paginationSelectLabels`. Should be changed to `paginationSelects`and `rowsSelected` respectively.

Comment (armstrmi) [reply ui/src/pages/overview/OverviewPage.tsx:149]: nit: also might be helpful to have a comment above each for loop to tell which controls are being emptied/hidden. Some of these class names can be a little difficult to find/comprehend.

ex:
//removes any datagrid drop down controls
*** for loop ***

Comment (shazlehu) [reply ui/src/pages/overview/OverviewPage.tsx:149]: Dang, that was sloppy of me -- This is exactly what code review is for!

**Review by armstrmi — APPROVED** [review]

File: ui/src/pages/overview/OverviewPage.tsx (hunk 1 above)
Comment (shazlehu) [reply ui/src/pages/overview/OverviewPage.tsx]: All fixed

**Pull request merged**